### PR TITLE
MEI support: adjust to new laissez vibrer

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR updates the MEI support to adapt the newly introduced laissez vibrer ties.

### CODE REVIEW

**Comment by rettinghaus** [comment]
Pinging @lpugin and @miiizen

**Review by lpugin — APPROVED** [review]
Looks good to me, thanks!

File: src/importexport/mei/internal/meiconverter.cpp [hunk 1]
```diff
@@ -1707,50 +1708,25 @@ libmei::Harm Convert::harmToMEI(const engraving::Harmony* harmony, StringList& m
...
-        break;
-    case (libmei::curvature_CURVEDIR_below):
-        lv->setAnchor(engraving::ArticulationAnchor::BOTTOM);
-        break;
-    default:
-        lv->setAnchor(engraving::ArticulationAnchor::AUTO);
-        break;
+    if (meiLv.HasCurvedir()) {
+        lv->setSlurDirection(Convert::curvedirFromMEI(meiLv.GetCurvedir(), warning));
+        //slur->setPropertyFlags(engraving::Pid::PLACEMENT, engraving::PropertyFlags::UNSTYLED);
```
Comment (miiizen) [inline src/importexport/mei/internal/meiconverter.cpp:1718]: This can be deleted

File: src/importexport/mei/internal/meiconverter.h [hunk 2]
```diff
@@ -226,8 +226,8 @@ class Convert
     static void layerIdentFromMEI(engraving::EngravingItem* item, const libmei::Element& meiElement);
     static void layerIdentToMEI(const engraving::EngravingItem* item, libmei::Element& meiElement);
 
-    static void lvFromMEI(engraving::Articulation* lv, const libmei::Lv& meiLv, bool& warning);
-    static libmei::Lv lvToMEI(const engraving::Articulation* lv);
+    static void lvFromMEI(engraving::LaissezVib* lv, const libmei::Lv& meiLv, bool& warning);
+    // static libmei::Lv lvToMEI(const engraving::LaissezVib* lv);
```
Comment (miiizen) [inline src/importexport/mei/internal/meiconverter.h:230]: This can be deleted

**Review by miiizen — COMMENTED** [review]
Looks good to me! There are just 2 comments to remove.

**Pull request merged**